movement/plots/trajectory.py

```diff
@@ -0,0 +1,114 @@
+"""Wrappers to plot movement data."""
+
+import xarray as xr
+from matplotlib import pyplot as plt
+
+DEFAULT_PLOTTING_ARGS = {
+    "s": 15,
+    "marker": "o",
+    "alpha": 1.0,
+}
+
+
+def plot_trajectory(
+    da: xr.DataArray,
+    keypoints: str | list[str] | None = None,
+    individual: str | None = None,
```

Comment:
Last point (I'd missed that before): please reorder the arguments to be `da, individual, keypoints, ax` to match the order in the docstring.
Or change the order of arguments in the docstring. In any case, the order should match in function signature and docstring.